apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java

```diff
@@ -494,6 +496,7 @@ public static FissClaim.Builder createDefaultClaimBuilder() {
           .setCurrStatusEnum(FissClaimStatus.CLAIM_STATUS_ROUTING)
           .setCurrTranDtCymd(HARDCODED_TRAN_DATE_CYMD)
           .setFedTaxNb(HARDCODED_FED_TAX_NUMBER)
+          .setIntermediaryNb(HARDCODED_INTERMEDIARY_NB)
```

Comment:
Do we need to set the claim key too so they can be properly ingested later?

Reply:
Good point.  I'll add that to the TODO list for PACA-752 when we switch to using it.